src/storage/txn/latch.rs

```diff
@@ -63,7 +62,7 @@ impl Lock {
 /// Each latch is indexed by a slot ID, hence the term latch and slot are used interchangeably, but
 /// conceptually a latch is a queue, and a slot is an index to the queue.
 pub struct Latches {
-    slots: Vec<Latch>,
+    slots: Vec<Mutex<Latch>>,
```

Comment:
here I just think that if we use SpinLock, can we gain a better performance?

Reply:
we can do a benchmark later @hicqu

Reply:
sure.